### PR TITLE
Add log file formats doc from old mavlink site

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

**Comment by hamishwillee** [comment]
@DonLakeFlyer Is this one OK to merge?

**Comment by hamishwillee** [comment]
@DonLakeFlyer Is this one OK to merge?

File: en/file_formats/mavlink.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Log File Formats
+
+*QGroundControl* supports two log file formats:
+
+* Plain MAVLink packet logs to "replay" a previous mission.
+* Comma-separated values (CSV) for further analysis and visualization (e.g. into Matlab, R, Mathematica, etc)
```
Comment (DonLakeFlyer) [inline en/file_formats/mavlink.md]: As far as I know CSV is not supported

**Comment by DonLakeFlyer** [comment]
Other than CSV looks fine

File: en/file_formats/mavlink.md (hunk 1 above)
Comment (hamishwillee) [reply en/file_formats/mavlink.md]: I'll remove that then and merge.  Thanks!

**Comment by hamishwillee** [comment]
OK, removed the mention of CSV, and associated "Data Log Format" and section on "The data log format is used when logging from within the Linechart view."

**Pull request merged**